mautic/mautic
# Fix: Pipedrive multiselect field sync

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/251
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5163
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This bugfix/feature has been sponsored by @Webmecanik

When you try to sync. a select type field, the fields are mapped but not synced.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create new Custom multiselect field in Pipedrive with three optiosn (first, second, third)
2. Create new Custom multiselect field in Mautic with the same options = labels and values should be same as label at Pipedrive
3. Setup Pipedrive plugin and match Custom multiselect field from  Pipedrive to Mautic
4.  Create form with Multiselect field and match to Contact same contact field
5: Add Push contact to integration  action
6. Send form and see this field cannot be added to Pipedrive

#### Steps to test the bug:
1. Repeat steps
2. You should see update field in Pipedrive

![image](https://user-images.githubusercontent.com/462477/32239877-0edf4d74-be6c-11e7-9746-e91c95578836.png)
